ecommerce/api.py

```diff
@@ -749,31 +749,31 @@ def get_available_bulk_product_coupons(coupon_payment_id, product_id):
     )
 
 
-def bulk_assign_product_coupons(emails, available_product_coupons):
+def bulk_assign_product_coupons(desired_assignments, bulk_assignment=None):
     """
     Assign product coupons to emails in bulk and create a record of this bulk creation
 
     Args:
-        emails (iterable of str):
-        available_product_coupons (iterable of CouponEligibility):
+        desired_assignments (iterable of (str, int)): An iterable of emails paired with the
+            ProductCoupon id that each email should be assigned
+        bulk_assignment (BulkCouponAssignment or None): A BulkCouponAssignment object, or
+            None if a new one should be created
 
     Returns:
-        iterable of (str, CouponEligibility): An iterable of tuples, where each one
-            is an email paired with the product coupon assigned to that email.
+        list of ProductCouponAssignment: The created ProductCouponAssignments
```

Comment:
Looks like it returns a (BulkAssignment, list of ProductCouponAssignment) tuple?